src/controllers/messages/Createmessage.js

```diff
@@ -1,15 +1,7 @@
 import moment from 'moment';
 import messages from '../../model/messages';
-import { validateMessage } from '../../helpers/validations/message';
 
-export const apiCreateMessage = (req, res, next) => {
-  const { error } = validateMessage(req.body);
-  if (error) {
-    return res.status(400).send({
-      status: 400,
-      error: error.details[0].message
-    });
-  }
+export const CreateMessage = (req, res) => {
```

Comment:
According to Airbnb style guide for naming convention, `Use camelCase when naming objects, functions, and instances.`, `Use PascalCase only when naming constructors or classes.`
https://github.com/airbnb/javascript#naming-conventions.

Basically means function names should be camelCase.